tastytrade/order.py

```diff
@@ -269,16 +295,10 @@ class ComplexOrder(TastytradeJsonDataclass):
     """
     Dataclass containing information about a complex order.
     """
-    id: str
```

Comment:
I believe these lines are needed for previously placed complex orders, you can confirm by calling `Account.get_history` after placing a complex order. If so, this could be renamed to `PlacedComplexOrder` to maintain the naming convention.

Reply:
These lines are no longer in the return value of placing complex orders. If there are other complex orders than OTOCO and OCO, then maybe, but those 2 are the only one I found.

Reply:
Check the value after calling `Account.get_history` with a complex order in the history, that's why this is there in the first place

Reply:
Looks like it's for pairs trades!